### DISCUSSION TITLE
Diagrams not displaying data in Version 22.2.4.202211062135

### DISCUSSION THREAD

**Discussion by dmcgivney1** [dbeaver/dbeaver#18214] [General]
I updated to the latest version from Microsoft store , DBeaver CE ver  , see above.  The Diagram window that used to show the table and foreign keys not longer displayes the Columns or any  associated tables connected by foreign key.
 
![Capture](https://user-images.githubusercontent.com/73199430/201708867-0ede72da-ad8f-4dc1-8840-4f11444e501d.PNG)

**Comment by dmcgivney1** [comment]
I want to follow that up:

Platform Windows 10.  
connected to Sybase ASE 12.5

**Comment by TheGuern** [comment]
I confirm that the er table diagrams are missing most of their vital data, including column names, data types, nullability, foreign keys, etc.

**Reply by TheGuern** [reply]
(same version and environment settings)

**Reply by TheGuern** [reply]
dBeaver finally filled in the column info in my er diagram. I just left the program running for a couple hours. When I re-opened the window the diagram had been updated WITH THE COLUMN DATA. I then add a new table, and it is still empty. So, it is not completely broken…

**Comment by E1izabeth** [comment]
Thank you for reporting.
Could you send the log files? You can find instructions here https://dbeaver.com/docs/wiki/Log-files/

**Reply by dmcgivney1** [reply]
I opened DBeaver today and loaded a table diagram.  It appears to now be working and displaying the diagrams again along with all supporting data. I t has been a few days and I have used Deaver for several hours and restarted my systems a few times.  At some point after the update, maybe after a restart, the diagrams appear to be working again.
thanks for your quick response
